src/hotspot/share/utilities/resourceHash.hpp

```diff
@@ -197,5 +212,39 @@ class ResourceHashtable : public ResourceObj {
   }
 };
 
+template<unsigned TABLE_SIZE, typename K, typename V>
+class FixedResourceHashtableStorage : public ResourceObj {
+  using Node = ResourceHashtableNode<K, V>;
+
+  Node* _table[TABLE_SIZE];
+protected:
+  FixedResourceHashtableStorage() {
+    memset(_table, 0, TABLE_SIZE * sizeof(Node*));
```

Comment:
Instead of memset, consider `FixedResourceHashtableStorage() : _table{} {}`.

Reply:
Fixed.